contrib/docker/testnet.sh

```diff
@@ -74,6 +100,19 @@ set_repo()
   fi
 }
 
+set_bins()
+{
+  read_bool_input "Use binaries from repo?" KOVRI_USE_REPO_BINS ""
```

Comment:
Can you add a note to the question (or somewhere) about ensuring that the binaries are built statically if not built within the container?